### PR TITLE
disbands badger force, turns it into general badger duty, and add urgent requests blurb

### PR DESCRIPTION
**Pull request by danchapman**
- Updates the badger duty description
- Adds urgent request blurb

### CODE REVIEW

File: working_agreement/README.md [hunk 1]
```diff
@@ -10,14 +10,23 @@ A living document for our ever evolving working agreements.
...
+- Unless a special case is made, no more than 25% of a dev's time should be spent on HB's
+
+
+## Urgent Requests
+
+- Given the nature of our business, small time sensitive tasks come up during the course of the day
+- Dev managers are responsible for assessing the urgency and priority of these tasks.
+- Dev managers rotate these tasks so that no one developer has to shoulder them.
+- Dev managers do their best to distrubute these tasks at standup.
+- A dev should be prepared to hand off at least one, no more than two, of these small requests per day.
```
Comment (jmdeldin) [inline working_agreement/README.md]: This line is confusing. Should this be "dev manager" or should we change "hand off" to "complete"?

File: working_agreement/README.md [hunk 2]
```diff
@@ -10,14 +10,23 @@ A living document for our ever evolving working agreements.
...
+- In the event of a critical problem, any devs should be willing to stop what they are working on to resolve the issue.
```
Comment (jmdeldin) [inline working_agreement/README.md]: :s/any/all/?

File: working_agreement/README.md [hunk 3]
```diff
@@ -10,14 +10,23 @@ A living document for our ever evolving working agreements.
...
-- Each member spends 20 hours a week dedicated to Badger Force related tasks.
-- The two members pair and call in the back ups as needed.
-- Members are not assigned _randos_, they focus on Honey Badgers and bugs. 
-
-
+## Badger Duty
+
+- Honeybadgers are assigned during Storytime
+- Each server-side dev is assigned an HB for the week, and it it's that person's job to resolve it
+- Front-end devs pair with a server-side dev on their HB
```
Comment (jmdeldin) [inline working_agreement/README.md]: This isn't true anymore, right?

File: working_agreement/README.md [hunk 4]
```diff
@@ -10,14 +10,23 @@ A living document for our ever evolving working agreements.
...
+- If you finish your HB quickly, you can let the team know so that those who are stuck can reach out for help.
+- If you know who can easily fix the HB, you can pair with that person, but you are still responsible for seeing it through.
+- If it takes longer than a week to fix, it is reviewed at the next Storytime.
+- It can be turned into a larger project, or reassigned for the week.
```
Comment (jmdeldin) [inline working_agreement/README.md]: Remove the "larger project" clause? Or is that something we have yet to do?

**Comment by jmdeldin** [comment]
You've got some Vim turd files checked in (.swo and .swp), and outside of my minor comments, this looks good. Thanks for putting this together!

**Comment by civrot** [comment]
@chappy can you clean out the vim files?

**Comment by civrot** [comment]
@chappy?

**Comment by danchapman** [comment]
crap, yeah on it

**Pull request merged**